force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -259,12 +294,21 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Integer performDivisionAndCast(Double a, Double b) {
         // Perform the division and cast (round down) off the result.
-        Double divisionResult = null;
+        Double divisionResult = a/b;
+        
 
         // Write the code for type casting the divisionResult to an Integer
         Integer roundedResult = null;
-        return null; // Replace null with the variable you used to store the result
+        roundedResult = divisionResult.intValue();
```

Comment:
Nice use of intValue(), for extra practice, try casting the divisionResult into an integer